### PR TITLE
add landing message #1

### PR DESCRIPTION
**Pull request by Dirty-Duski**
added a header to index.html

### CODE REVIEW

**Comment by BlueEyeDanger** [comment]
not sure about the <h1>

File: index.html [hunk 1]
```diff
@@ -1 +1,2 @@
+<h1>Header</h1>
```
Comment (BlueEyeDanger) [inline index.html:1]: im not sure about this code.

Comment (Dirty-Duski) [reply index.html:1]: I'm answering your comment and resolving the conversation

**Review by BlueEyeDanger — APPROVED** [review]

**Review by BlueEyeDanger — CHANGES_REQUESTED** [review]
requesting changes here

**Review by BlueEyeDanger — APPROVED** [review]

**Pull request merged**